packages/aws-event-stream/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[project]
+name = "aws-event-stream"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+authors = [
+    { name = "Hayden Baker", email = "haydebak@amazon.com" }
+]
```

Comment:
Bold, but not what we want here. iirc this is optional so best to remove it

Reply:
Do we want to add any other fields from the previous toml files? e.g. `license`, `keywords`, `classifiers`, etc?

Reply:
Yeah let's add those, but as a separate PR so that this can get merged